d3ploy/demand_driven_deployment_inst.py

```diff
@@ -31,139 +31,141 @@ class DemandDrivenDeploymentInst(Institution):
     """
 
     facility_commod = ts.MapStringString(
-        doc="A map of facilities and each of their corresponding" +
-        " output commodities",
-        tooltip="Map of facilities and output commodities in the " +
-        "institution",
-        alias=['facility_commod', 'facility', 'commod'],
-        uilabel="Facility and Commodities"
+        doc="A map of facilities and each of their corresponding"
+        + " output commodities",
+        tooltip="Map of facilities and output commodities in the "
+        + "institution",
+        alias=["facility_commod", "facility", "commod"],
+        uilabel="Facility and Commodities",
     )
 
     facility_capacity = ts.MapStringDouble(
-        doc="A map of facilities and each of their corresponding" +
-        " capacities",
-        tooltip="Map of facilities and capacities in the " +
-        "institution",
-        alias=['facility_capacity', 'facility', 'capacity'],
-        uilabel="Facility and Capacities"
+        doc="A map of facilities and each of their corresponding"
+        + " capacities",
+        tooltip="Map of facilities and capacities in the " + "institution",
+        alias=["facility_capacity", "facility", "capacity"],
+        uilabel="Facility and Capacities",
     )
 
     facility_pref = ts.MapStringString(
-        doc="A map of facilities and each of their corresponding" +
-        " preferences",
-        tooltip="Map of facilities and preferences in the " +
-        "institution",
-        alias=['facility_pref', 'facility', 'pref'],
+        doc="A map of facilities and each of their corresponding"
+        + " preferences",
+        tooltip="Map of facilities and preferences in the " + "institution",
+        alias=["facility_pref", "facility", "pref"],
         uilabel="Facility and Preferences",
-        default={}
+        default={},
```

Comment:
why this comma?

Reply:
I tested it with and without the comma, and the `pytests` all pass regardless of these commas.

Reply:
I reverted all pep8 changes